### PR TITLE
Idn improvements

### PR DESCRIPTION
**Pull request by drwetter**
see each comment

### CODE REVIEW

File: testssl.sh [hunk 1]
```diff
@@ -17384,11 +17425,26 @@ get_local_a() {
...
 }
 
-# does a hard exit if no lookup binary is provided
+# Does a hard exit if no lookup binary is provided
+# Checks for IDN capabilities also
+#
 check_resolver_bins() {
-     if ! type -p dig &> /dev/null && ! type -p host &> /dev/null && ! type -p drill &> /dev/null && ! type -p nslookup &>/dev/null; then
+     type -p dig   &> /dev/null &&  HAS_DIG=true
+     type -p dig   &> /dev/null &&  HAS_HOST=true
```
Comment (teward) [inline testssl.sh]: Think you meant this to be `host`

Comment (drwetter) [reply testssl.sh]: done

File: testssl.sh [hunk 2]
```diff
@@ -355,7 +355,16 @@ HAS_CHACHA20=false
...
+HAS_DIG=false
+HAS_HOST=false
+HAS_DRILL=false
+HAS_NSLOOKUP=false
+HAS_IDN=false
+HAS_IDN2=false
+HAS_AVAHIRESOLVE=false
+HAS_DIG_NOIDNOUT=false
+
+ OSSL_CIPHERS_S=""
```
Comment (teward) [inline testssl.sh]: Style: Extra space in the beginning is probably not supposed to be here, so it's all on one line.  Just being nitpicky at this point, but still.

Comment (drwetter) [reply testssl.sh]: :-)

done

**Pull request merged**